apps/web-giddh/src/app/new-inventory/component/create-update-group/create-update-group.component.ts

```diff
@@ -184,6 +191,20 @@ export class CreateUpdateGroupComponent implements OnInit, OnDestroy {
         });
     }
 
+    /**
+    * Get all discounts api call
+    *
+    * @private
+    * @memberof CreateUpdateGroupComponent
+    */
+    private getAllDiscounts(): void {
+        this.settingsDiscountService.GetDiscounts().pipe(takeUntil(this.destroyed$)).subscribe(response => {
+            if (response?.body?.length) {
```

Comment:
move to component store 
confirm with @ravindersinghdang

Reply:
Done